pkgs/data/icons/antu-icon-theme/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenvNoCC, fetchFromGitLab }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "antu-icon-theme";
+  version = "0-unstable-2024-01-23";
+
+  src = fetchFromGitLab {
+    owner = "froodo_alexis";
+    repo = "antu-icons";
+    rev = "279be277";
+    hash = "sha256-CLcr+X/b0moVEBV0O/dzCDq4w5G2+KRLUBdqKm0eAKA=";
+  };
+
+  dontDropIconThemeCache = true;
+
+  installPhase = ''
+    mkdir -p $out/share/icons/antu-icons
+    cp -r * $out/share/icons/antu-icons
+  '';
+
+  meta = with lib; {
+    description = "A smooth icon theme designed for Plasma Desktop";
+    homepage = "https://gitlab.com/froodo_alexis/Antu-icons";
+    license = licenses.cc-by-nc-sa-30;
+    platforms = platforms.linux;
```

I think icons can be useful anywhere, even though things like `index.theme` might target linux-based graphical systems.

```suggestion
    platforms = platforms.all;
```